samples/cellular/nrf_cloud_multi_service/overlay_nrf7002ek_wifi_no_lte.conf

```diff
@@ -95,7 +96,7 @@ CONFIG_LOCATION_TRACKING_WIFI=y
 CONFIG_LOCATION_METHOD_WIFI=y
 # Align this with CONFIG_LOCATION_METHOD_WIFI_SCANNING_RESULTS_MAX_CNT.
 # Also see comments for CONFIG_HEAP_MEM_POOL_SIZE.
-# We set this to 30 in this overlay (as opposed to 60 in the wifi-scan-only overlay) so that
+# We set this to 30 in this overlay so that
```

Comment:
@glarsennordic I don't know what you'd like to have here but 60 is wrong as it's currently 20 in overlay-nrf7002ek-wifi-scan-only.conf

Reply:
> If you wind up rebasing, could you set this to 20?

So you want `CONFIG_WIFI_MGMT_SCAN_MAX_BSS_CNT=20`?

Reply:
Yes, but don't bother with it unless you are already making changes to the area, we can always change it ourselves when we next adjust configs. Just making the comment accurate (as you have done) is just fine

Reply:
Ok, I was about to change this already but I guess `CONFIG_HEAP_MEM_POOL_SIZE` comment and maybe even the value should be tuned so I'll leave it to you.